README.md

```diff
@@ -1,7 +1,17 @@
-# TiKV
+![tikv_logo](images/logo.png)
 
 [![Build Status](https://circleci.com/gh/tikv/tikv.svg?style=shield&circle-token=36bab0a8e43edb0941b31c38557d2d9d0d58f708)](https://circleci.com/gh/tikv/tikv) [![Coverage Status](https://codecov.io/gh/tikv/tikv/branch/master/graph/badge.svg)](https://codecov.io/gh/tikv/tikv) ![GitHub release](https://img.shields.io/github/release/tikv/tikv.svg)
 
+
+TiKV is an open source distributed Key-Value storage system; Unlike other traditional NoSQL systems, TiKV not only provides classical Key-Value style APIs, but also ACID semantic APIs.
+
+The design of TiKV is deeply influenced by some great distributed systems by Google, including BigTable, Spanner and Percolator, combined some of the latest achievements in academia in recent year, e.g. Raft consensus algorithm;
```

Comment:
-> The design of TiKV is inspired by some great distributed systems by Google, such as BigTable, Spanner, and Percolator, and some of the latest achievements in the academia in recent years, such as the Raft consensus algorithm.